python/pyspark/ml/feature.py

```diff
@@ -65,7 +65,8 @@
 
 
 @inherit_doc
-class Binarizer(JavaTransformer, HasInputCol, HasOutputCol, JavaMLReadable, JavaMLWritable):
+class Binarizer(JavaTransformer, HasThreshold, HasThresholds, HasInputCol, HasOutputCol,
+                HasInputCols, HasOutputCols, JavaMLReadable, JavaMLWritable):
```

Comment:
Since multi-column is supported now, maybe change the below docstring to reflect the multi-column support? And also change the Scala doc?